app/forms/supplementary/AddressAndIdentification.scala

```diff
@@ -81,4 +81,6 @@ object AddressAndIdentification {
       "declaration.declarant.address.countryCode" ->
         allCountries.find(country => country.countryName == address.country).map(_.countryCode).getOrElse("")
     )
+
+  def toConsigneeMetadataProperties(address: AddressAndIdentification): Map[String, String] = ???
```

Comment:
We don't use this method, so it's not a problem to have `???` here, it's waiting for xml mapping